src/hooks/useSwap.ts

```diff
@@ -257,303 +239,292 @@ export const useSwap = ({ accountId, selector }: Props) => {
     inputs: CallRequestIntentProps,
     mutate?: (input: CallRequestIntentProps) => void
   ): Promise<NearTX[] | void> => {
-    try {
-      if (
-        !isValidInputs(inputs) &&
-        !isValidEstimateQueue(inputs?.estimateQueue)
-      )
-        return
-      const {
-        tokenIn,
-        tokenOut,
-        selectedTokenIn,
-        selectedTokenOut,
-        clientId,
-        estimateQueue,
-      } = inputs
-
-      setIsProcessing(true)
-
-      const unitsSendAmount = parseUnits(
-        tokenIn,
-        selectedTokenIn?.decimals as number
-      ).toString()
-      const estimateUnitsBackAmount = parseUnits(
-        tokenOut,
-        selectedTokenOut?.decimals as number
-      ).toString()
-      const getBlock = await getNearBlock()
-      const msg = {
-        CreateIntent: {
-          id: clientId,
-          IntentStruct: {
-            initiator: accountId,
-            send: {
-              token_id:
-                selectedTokenIn!.address === "native"
-                  ? "wrap.near"
-                  : selectedTokenIn!.address,
-              amount: unitsSendAmount,
-            },
-            receive: {
-              token_id:
-                selectedTokenOut!.address === "native"
-                  ? "wrap.near"
-                  : selectedTokenOut!.address,
-              amount: estimateUnitsBackAmount,
-            },
-            expiration: {
-              Block: getBlock.height + CREATE_INTENT_EXPIRATION_BLOCK_BOOST,
-            },
-            referral: {
-              Some: REFERRAL_ACCOUNT,
-            },
+    if (!isValidInputs(inputs) && !isValidEstimateQueue(inputs?.estimateQueue))
```

Comment:
I don't wrap the utility function into into try/catch , because the  reaction to it should be handled in UI component, unless there is an error based logic.